install.gup

```diff
@@ -0,0 +1,8 @@
+#!bash -eu
+here="$(pwd)"
+gup --always
+
+dest="$PREFIX/usr/share/gnome-shell/extensions/shellshape@gfxmonk.net"
```

Comment:
This will fail if `PREFIX` isn't set, since I use `-u` in the shebang line. `"${PREFIX:-}/usr/..."` should work